Sources/BraveWallet/Crypto/Stores/WalletStore.swift

```diff
@@ -29,7 +29,8 @@ public class WalletStore {
     blockchainRegistry: BraveWalletBlockchainRegistry,
     txService: BraveWalletTxService,
     ethTxManagerProxy: BraveWalletEthTxManagerProxy,
-    solTxManagerProxy: BraveWalletSolanaTxManagerProxy
+    solTxManagerProxy: BraveWalletSolanaTxManagerProxy,
+    ipfsApi: IpfsAPI?
```

Comment:
Is there a scenario where `IpfsAPI` would actually be nil? 
If not, it might make more sense to make this non-optional within `Sources/BraveWallet` (in `CryptoStore`, `WalletStore`, etc.) and guard against it in `BrowserViewController+Wallet.swift` helpers for initializing `WalletStore` and `CryptoStore`: https://github.com/brave/brave-ios/pull/7000/files#diff-aec1409b3eca46029a30cae384586f7e31b96a567d0fc78bdf53d74361a7ec64R20-R22

Reply:
This is needed for tests since as i understand there is no good way to mock IpfsApi